nevergrad/optimization/optimizerlib.py

```diff
@@ -1561,6 +1588,190 @@ def _internal_tell_not_asked(self, candidate: base.Candidate, value: float) -> N
         raise base.TellNotAskedNotSupportedError
 
 
+class EMNA_TBPSA(TBPSA):
```

Comment:
Please use the updated version of TBPSA as example, this now fails because of rounding errors when using the array as a key for a dictionary

Reply:
https://github.com/facebookresearch/nevergrad/commit/865938ff8a173773372f842722bf61c62eb47980

Reply:
done!